source/documentation/monitoring-an-app/how-to-create-alarms.html.md.erb

```diff
@@ -103,23 +109,31 @@ kubectl -n <namespace> describe prometheusrules prometheus-custom-rules-<applica
 ```
 
 ### PrometheusRule examples
+  
 If you're struggling for ideas on how and which alerts to setup, please see some examples [here](https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/master/terraform/cloud-platform-components/resources/prometheusrule-examples/application-alerts.yaml).
 
-### Advisory Note 1: PrometheusRules status incase of DR/requirement for a new Prometheus Install
+### Advisory Note 1: If Prometheus gets re-installed
 
-The  `PrometheusRule` is a [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/##customresourcedefinitions) that declaratively defines a desired Prometheus rule to be consumed by Prometheus and applied using a YAML file. However, if for any reason Prometheus has to be uninstalled, `all PrometheusRules are removed with the CRD.`
+In the event of a serious failure, or for a required upgrade, it may 
```

Comment:
@digitalronin , should it say, it may.. "remove your prometheus rules"?

Reply:
Oops. Good catch. I'll fix that.